src/locales/en_US.ts

```diff
@@ -1931,6 +1933,20 @@ const strings = {
   see_all: 'See All',
   sell_crypto: 'Sell Crypto',
   sell_crypto_footer: 'Crypto to bank or cash',
+
+  // Trade Option Select Scene
+  trade_option_buy_title: 'Buy Cryptocurrency',
+  trade_option_sell_title: 'Sell Cryptocurrency',
+  trade_option_select_payment_method: 'Select Payment Method',
+  trade_option_choose_provider: 'Choose Provider',
+  trade_option_no_quotes_title: 'No quotes available',
+  trade_option_no_quotes_body:
+    'Please try again later. No providers are currently available.',
+  trade_option_total_label: 'Total',
+  trade_option_settlement_label: 'Settlement',
+  trade_option_powered_by_label: 'Powered By:',
```

Comment:
Use `plugin_powered_by_space` I'm sure things will look fine without the `:`

Reply:
Remove the `":"` character from the lstring value